ext/node/polyfills/testing.ts

```diff
@@ -4,11 +4,45 @@ import { primordials } from "ext:core/mod.js";
 const {
   PromisePrototypeThen,
   ArrayPrototypePush,
+  ArrayPrototypeForEach,
   SafePromiseAll,
   SafePromisePrototypeFinally,
 } = primordials;
 import { notImplemented, warnNotImplemented } from "ext:deno_node/_utils.ts";
 
+const methodsToCopy = [
+  "deepEqual",
+  "deepStrictEqual",
+  "doesNotMatch",
+  "doesNotReject",
+  "doesNotThrow",
+  "equal",
+  "fail",
+  "ifError",
+  "match",
+  "notDeepEqual",
+  "notDeepStrictEqual",
+  "notEqual",
+  "notStrictEqual",
+  "partialDeepStrictEqual",
+  "rejects",
+  "strictEqual",
+  "throws",
+];
+
+/** `assert` object available via t.assert */
+let assertObject = undefined;
+function getAssertObject() {
+  if (assertObject === undefined) {
+    const assert = process.getBuiltinModule("node:assert");
```

Comment:
Shouldn't this be a direct import of `assert` module - since we use ESM instead of CJS I don't think there's a need for going through `process.getBuiltinModule`

Reply:
That seems to work. Updated